source/rules/roles.txt

```diff
@@ -176,12 +183,27 @@ its fields, App Services omits the document or fields from the results.
 How App Services Assigns Roles
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-App Services dynamically assigns roles for every request. For each document 
-that matches the incoming query, the user is either assigned a separate role 
-or no role.
+App Services assigns roles at different times depending on whether you're using
+:ref:`Device Sync (Flexible Mode) <sync>` or not.
```

Comment:
This section only deals with Flexible Sync, correct? The Partition-Based Sync stuff is covered elsewhere.

I think we could be clearer here by replacing most instances of "Device Sync" with "Flexible Sync". This makes it clearer that this doesn't work for Partition-Based Sync.

Reply:
It's tricky because I'm trying to subtly make "Device Sync" mean "Flexible Sync", as it should eventually supersede PBS.

Reply:
Ah, I see. I think we run the risk of confusing some people, but the number is probably pretty small. Might be worth it to gradually shift Device Sync perception the direction you're going.